services/movie.py

```diff
@@ -0,0 +1,44 @@
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if actors_ids:
+        queryset = queryset.filter(
+            actors__id__in=actors_ids,
+        )
+
+    if genres_ids:
+        queryset = queryset.filter(
+            genres__id__in=genres_ids,
+        )
+
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> QuerySet:
```

Comment:
Sorry, missed this one: this function also create a `Movie` instance, not `QuerySet`.